Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.3
+// swift-tools-version: 5.4
```

Comment:
While the whitespace doesn't matter in 5.4, please remove the new space.

Reply:
Actually, the [doc](https://developer.apple.com/documentation/packagedescription/package) says that from 5.4 we have to add a whitespace

Reply:
> Version 5.4 and later has relaxed the whitespace requirements. 

The parser is more flexible with 5.4; it does not require a leading whitespace, but does allow it. 

I also read the original 5.4 release notes, and use SPM extensively on several projects.

Reply:
Perfect! Thanks for the info 🚀